src/suricata.c

```diff
@@ -1809,6 +1825,45 @@ static TmEcode ParseCommandLine(int argc, char** argv, SCInstance *suri)
                 }
             }
 #endif
+            else if(strcmp((long_opts[option_index]).name, "windivert-forward") == 0) {
```

Comment:
I don't think the difference between windivert and windivert-forward is documented?

Reply:
It's explained in the userdoc like using the forward layer of NFQ, but maybe I can make it clearer in the usage printout.